sickbeard/providers/extratorrent.py

```diff
@@ -50,6 +50,7 @@ def __init__(self):
         self.custom_url = None
 
         # Proper Strings
+        self.proper_strings = ['PROPER', 'REPACK', 'REAL']
 
         # Miscellaneous Options
         self.headers.update({'User-Agent': USER_AGENT})
```

Comment:
I just kept it from the old version. If it's not needed anymore, I'll remove it.